source/blog/041-import-is-deprecated.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "`@import` is Deprecated"
+author: Jennifer Thakar
+date: 2024-07-18 17:00:00 -8
+---
+
+Back in 2019, we [released the Sass module system], adding new `@use` and
+`@forward` rules to the language that were designed to replace `@import` and
+make stylesheets more maintainable and less error-prone. We can now announce
+that `@import` is officially deprecated as of Dart Sass 1.78.0.
```

Comment:
I think it's worth at least a paragraph here talking up the benefits of `@use` and explaining why `@import` makes everything worse. It's not necessarily going to convince people, but it'll help set the stage for "we have thought about this and are making an informed decision, not just taking away your toys out of spite".